presto-product-tests/conf/presto/etc/catalog/sqlserver.properties

```diff
@@ -0,0 +1,4 @@
+connector.name=sqlserver
+connection-url=jdbc:sqlserver://sqlserver
+connection-user=sa
```

Comment:
We don't have that provision. Microsoft is working on that
https://github.com/Microsoft/mssql-docker/issues/18